content/06.discussion.md

```diff
@@ -54,6 +54,54 @@ In assessing the upper bound on the predictive performance of a deep learning mo
+Aleatoric uncertainty can be represented well by designing an appropriate loss function.
+In the case of data-dependent aleatoric uncertainty, one can train the model to increase its uncertainty when it is incorrect due to noisy or missing data, and in the case of task-depedent aleatoric uncertainty, one can optimize for the best uncertainty parameter for each task [@tag:uncertainty_multi_task].
+Meanwhile, there are various methods for modeling epistemic uncertainty, which we outline in what follows.
+
+In classification tasks, confidence calibration is the problem of using classifier scores to predict class membership probabilities that match the true membership likelihoods.
+These membership probabilities can be used to assess the uncertainty associated with assigning the example to each of the classes.
+Guo et al. [@tag:guo_calibration] observed that contemporary neural networks are poorly calibrated and provided a simple recommendation for calibration: temperature scaling, a single parameter special case of Platt scaling [@tag:platt_scaling].
+In addition to confidence calibration, there is early work from Chryssolouris et al. [@tag:Chryssolouris1996_confidence] that described a method for obtaining confidence intervals with the assumption of normally distributed error for the neural network.
+More recently, Hendrycks and Gimpel discovered that incorrect or out-of-distribution examples usually have lower maximum softmax probabilities than correctly classified examples, allowing for effective detection of misclassified examples [@tag:out_dist_baseline].
+Liang et al. used temperature scaling and small perturbations to further separate the softmax scores of correctly classified examples and the scores of out-of-distribution examples, allowing for more effective detection [@tag:temp_out_dist].
+
+An alternative approach for obtaining principled uncertainty estimates from deep learning models is to use Bayesian neural networks.
+Deep learning models are usually trained to obtain the most likely parameters given the data.
+However, choosing the single most likely set of parameters ignores the uncertainty about which set of parameters (among the possible models that explain the given dataset) should be used.
+This sometimes leads to uncertainty in predictions when the chosen likely parameters produce high-confidence but incorrect results.
+On the other hand, the parameters of Bayesian neural networks are modeled as full probability distributions.
+This Bayesian approach comes with a whole host of benefits, including: better calibrated confidence estimates [@tag:ai_safety] and more robustness to adversarial and out-of-distribution examples [@tag:strong_adversary].
+Unfortunately, modeling the full posterior distribution for the model’s parameters given the data is usually computationally intractable.
```

Comment:
Do we want a new paragraph here?  Or at the next line?

Reply:
I wanted to have the introduction of Bayesian neural nets and test-time dropout together in the same paragraph, because they are closely related. A previous comment suggested this as well. However, I can separate them if you think it will be more readable.

Reply:
Sounds good, we can keep them together.